teammapper-backend/src/map/entities/mmpMap.entity.ts

```diff
@@ -11,35 +11,36 @@ import { MmpNode } from './mmpNode.entity'
 @Entity()
 export class MmpMap {
   @PrimaryGeneratedColumn('uuid')
-  id: string
+  id: string;
 
-  @Column({ type: 'timestamptz', nullable: true, default: () => 'now()' })
-  lastModified: Date
+  @Column({ type: 'timestamptz', nullable: true, default: () => 'CURRENT_TIMESTAMP' })
+  lastModified: Date | null = new Date();
```

Comment:
do we need `new Date()` here? `CURRENT_TIMESTAMP` is already setting the timestamp.

Reply:
As far as I know CURRENT_TIMESTAMP is for Postgres, so `lastModified` would be null by default if you created a new MmpNode() in Typescript.

Reply:
yes, the point is you leave it simply empty until its saved. Thats what the Postgres default is there for. If you assign a default in javascript, you never reach an empty field where the Postgres default comes into play. Why do we need a javascript default?